frontend/tests/acceptance/person-cv-export-test.js

```diff
@@ -0,0 +1,20 @@
+import { module, skip } from "qunit";
+import applicationPage from "frontend/tests/pages/application";
+import setupApplicationTest from "frontend/tests/helpers/setup-application-test";
+import { selectChoose } from "ember-power-select/test-support";
+
+module("Acceptance | person cv export", function(hooks) {
+  setupApplicationTest(hooks);
+
+  skip("should generate right url", async function(assert) {
+    // Go to the start page and select a user from the dropdown
+    await applicationPage.visitHome("/");
+    await selectChoose("#people-search", ".ember-power-select-option", 0);
+
+    // TODO
```

Comment:
In my view, it would be better to implement this spec in another story, because I worked now multiple hours on this test and the story points in the ticket are used up.